### DISCUSSION TITLE
Curious about machine learning techniques

### DISCUSSION THREAD

**Discussion by sjmsjmdsg** [github/codeql#8305] [Q&A]
I’m now looking at their documents, it says besides rules, it also uses deep learning/machine learning model, to find more security vulnerabilities - but I could not find any details regarding what kind of model it is using.

Is anyone using this? Can you please point me to some description of those models?

<img width="582" alt="c522536818bd5c7e9e7f0ab446dbfcb" src="https://user-images.githubusercontent.com/19368085/156251696-260924fc-6531-4723-a9e6-4e5c67cdd2e0.png">

**Comment by intrigus-lgtm** [comment]
Have a look at this blog post:
https://github.blog/2022-02-17-leveraging-machine-learning-find-security-vulnerabilities/
